OCP-Infrastructure/helm/prometheus/templates/crd.yaml

```diff
@@ -14,11 +14,13 @@ spec:
   additionalScrapeConfigs:
     name: prometheus-scrape-config
     key: scrape-config.yaml
-  volumes:
-    - name: prometheus-persistence
-      persistentVolumeClaim:
-        claimName: prometheus-pvc
-        readOnly: false
+  storage:
+    volumeClaimTemplate:
+      spec:
+        storageClassName: rook-ceph-block
+        resources:
+          requests:
+            storage: 15Gi
```

Comment:
15Gi is plenty. I think 10% would be enough.

Reply:
Storage shouldnt be an issue rn, we can try with 15Gi and see how long it lasts.

Although persisting the data somewhere outside prometheus altogether might be preferable in the long run.